more_itertools/tests/test_more.py

```diff
@@ -836,9 +836,9 @@ def test_collapse(self):
         l = [[1], 2, [[3], 4], [[[5]]]]
         self.assertEqual(list(mi.collapse(l)), [1, 2, 3, 4, 5])
 
-    def test_collapse_to_string(self):
-        l = [["s1"], "s2", [["s3"], "s4"], [[["s5"]]]]
-        self.assertEqual(list(mi.collapse(l)), ["s1", "s2", "s3", "s4", "s5"])
+    def test_collapse_to_string_and_bytes(self):
+        l = [["s1"], "s2", [["s3"], b"s4"], [[[b"s5"]]]]
+        self.assertEqual(list(mi.collapse(l)), ["s1", "s2", "s3", b"s4", b"s5"])
```

Comment:
`flake8` is complaining about this line:

```suggestion
        self.assertEqual(
            list(mi.collapse(l)), ["s1", "s2", "s3", b"s4", b"s5"]
        )
```

Reply:
fixed